src/main/java/ch/wisv/areafiftylan/users/model/UserExportDTO.java

```diff
@@ -0,0 +1,17 @@
+package ch.wisv.areafiftylan.users.model;
+
+import ch.wisv.areafiftylan.seats.model.Seat;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
+@AllArgsConstructor
+public class UserExportDTO {
+
+    private String email;
+    private String password_hash;
+    private Long userId;
+    private List<Seat> seat;
```

Comment:
Why not remove this here and add it as a separate export? An export to a `SeatExportDTO` with `Long userId` and `String seatString`?

Reply:
A User has a seat, why would you export that separately. It's all tied to the userId, which is the whole point of this endpoint right?

Reply:
True, but if we want to export more, it's more expandable. It looks a little sloppy and more thrown together now. We could use it this way, but it looks a bit low quality :stuck_out_tongue: